### ISSUE TITLE
模型训练效果的问题

### ISSUE THREAD

**Issue by cutecharmingkid** [PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-#2]
大家使用bert-bilstm-attention-crf训练出来的模型效果如何？我使用训练好的网络依然无法抽取实体，请问是我的训练中存在问题吗？

**Comment by PeiChangY** [comment]
你训练的是什么领域实体，你只能识别这个领域的实体，并不是训练效果的问题




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月15日(星期三) 中午12:46
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-] 模型训练效果的问题 (Issue #2)





 
大家使用bert-bilstm-attention-crf训练出来的模型效果如何？我使用训练好的网络依然无法抽取实体，请问是我的训练中存在问题吗？

**Comment by cutecharmingkid** [comment]
您好，我是使用您的数据集msra_data进行训练，我使用predict文件预测使用的语句是：习近平是中华人民共和国的主席。返回的预测结果为空。同时系统提示：UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at  ..\aten\src\ATen\native\TensorCompare.cpp:328.)
  score = torch.where(mask[i].unsqueeze(1), next_score, score)

![image](https://user-images.githubusercontent.com/102043624/225215363-309e8a7a-6da4-4e33-aac8-61139e1eb160.png)

**Comment by cutecharmingkid** [comment]
> 你训练的是什么领域实体，你只能识别这个领域的实体，并不是训练效果的问题
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-" ***@***.***&gt;; 发送时间:&nbsp;2023年3月15日(星期三) 中午12:46 ***@***.***&gt;; ***@***.***&gt;; 主题:&nbsp;[PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-] 模型训练效果的问题 (Issue #2) 大家使用bert-bilstm-attention-crf训练出来的模型效果如何？我使用训练好的网络依然无法抽取实体，请问是我的训练中存在问题吗？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;
您好，我是使用您的数据集msra_data进行训练，我使用predict文件预测使用的语句是：习近平是中华人民共和国的主席。返回的预测结果为空。同时系统提示：UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at ..\aten\src\ATen\native\TensorCompare.cpp:328.)
score = torch.where(mask[i].unsqueeze(1), next_score, score)

![image](https://user-images.githubusercontent.com/102043624/225223025-8be95dc4-6dc8-4983-8543-f1099b3b7752.png)
